### PR TITLE
Fly-link update to 0.4.0

### PR DESCRIPTION
**Pull request by Fly33**
+ region selection with the polygon draw tool
+ exclusion of portals with the marker draw tool
+ steady drawing (portals is not excluded from the drawing if they are
moved out of boundaries)

### CODE REVIEW

**Comment by modos189** [comment]
Looks great. I see no obstacles to merger after the corrective release that will be released the other day.

**Comment by johnd0e** [comment]
@Fly33 
Could you comment ideas of #203?
Would it be hard to implement?

File: plugins/fly-links.js [hunk 1]
```diff
@@ -221,32 +217,168 @@ window.plugin.flyLinks.updateLayer = function() {
...
+    }
+  }
+  
   $.each(edges, function(idx, edge) {
     drawLink(edge.a, edge.b, {
       color: '#FF0000',
       opacity: 1,
       weight: 1.5,
-      interactive: false,
+      clickable: false,
```
Comment (modos189) [inline plugins/fly-links.js]:
```suggestion
      interactive: false,
```

File: plugins/fly-links.js [hunk 2]
```diff
@@ -221,32 +217,168 @@ window.plugin.flyLinks.updateLayer = function() {
...
-  
+
   $.each(triangles, function(idx, triangle) {
     drawField(triangle.a, triangle.b, triangle.c, {
       stroke: false,
       fill: true,
       fillColor: '#FF0000',
       fillOpacity: 1 - Math.pow(0.85, triangle.depth),
-      interactive: false,
+      clickable: false,
```
Comment (modos189) [inline plugins/fly-links.js]:
```suggestion
      interactive: false,
```

**Review by modos189 — CHANGES_REQUESTED** [review]
New version of Leaflet uses the keyword *interactive*. Please do not return outdated *clickable*

**Review by modos189 — APPROVED** [review]

**Pull request merged**